forena.report.inc

```diff
@@ -1420,9 +1420,9 @@ function forena_report_add_parameter_form($form, &$form_state, $report_name) {
   );
 
   $form['cancel'] = array(
-      '#type' => 'submit',
-      '#value' => t('Cancel'),
-      '#submit' => array('forena_update_cancel'),
+       '#type'  => 'link',
```

Comment:
[Drupal coding standards](https://www.drupal.org/docs/develop/standards/coding-standards#indenting) specify two spaces per indent.  It is clear, however, that the author has not followed this specification consistently.  For example, compare 2 spaces here:
https://github.com/ulsdevteam/forena/pull/4/files#diff-354d443158c4e822b7b39bd734c6b6c1L1383
against 3 spaces here:
https://github.com/ulsdevteam/forena/pull/4/files#diff-354d443158c4e822b7b39bd734c6b6c1L1398

Since the upstream code is loose on the spec, I'll allow this.  We may get pushback when you submit this to the forena project, which is the next step.